### ISSUE TITLE
Log F1 scores in evaluation with sci-kit learn

### ISSUE THREAD

**Issue by varunsatish** [varunsatish/llama-recipes-prefer#13]

**Comment by msalganik** [comment]
I would like:

- log R^2_{eval}.  I know that we are not using R^2 in PreFer, but I have some intuition about it.
- log-loss https://scikit-learn.org/stable/modules/generated/sklearn.metrics.log_loss.html

**Comment by keyonvafa** [comment]
I think eval/loss is already log-loss. But it'd be a good sanity check to test the SKLearn implementation and make sure they're the same.

Iirc the code is set up so we have access to a list of predictions and a list of outcomes. From these two lists it'll be easy to iterate between many different binary classifications. I'll also add AUC as a useful metric.
